app/main.py

```diff
@@ -1,6 +1,20 @@
-# write your imports here
+from app.cinema.bar import CinemaBar
+from app.cinema.hall import CinemaHall
+from app.people.cinema_staff import Cleaner
+from app.people.customer import Customer
 
 
-def cinema_visit(customers: list, hall_number: int, cleaner: str, movie: str):
-    # write you code here
-    pass
+def cinema_visit(customers: list,
+                 hall_number: int,
+                 cleaner: str,
+                 movie: str) -> None:
+    customers_instances = [
+        Customer(customer["name"], customer["food"])
+        for customer in customers
+    ]
+    cinema_bar = CinemaBar()
+    for customer in customers_instances:
+        cinema_bar.sell_product(customer, customer.food)
```

Comment:
attention to whole project, when listening to comments.
```suggestion

    for customer in customers_instances:
        cinema_bar.sell_product(customer, customer.food)
        
```